examples/platformer/main.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"fmt"
+	"github.com/hajimehoshi/ebiten"
```

Comment:
Add a space line above this.

Reply:
This is not done? I don't want to bother you with doing such little thing, so I'll do.